src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -66,9 +66,9 @@
     <PackageReference Include="Microsoft.Azure.AppService.Proxy.Client" Version="2.0.11020001-fabe022e" />
     <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="1.0.3" />
     <PackageReference Include="Microsoft.Azure.Storage.File" Version="11.1.7" />
-    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.29" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Host.Storage" Version="4.0.3-11847" />
     <PackageReference Include="Microsoft.Azure.KeyVault" Version="3.0.3" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.30-11861" />
```

Comment:
These are test packages I've published to the app service test feed, to allow this PR to build and tests to run. Once the SDK PR is in, I'll update this to point to actual packages.